AllReadyApp/Web-App/AllReady/Areas/Admin/Features/EventManagerInvites/CreateEventManagerInviteCommandHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AllReady.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas.Admin.Features.EventManagerInvites
+{
+    public class CreateEventManagerInviteCommandHandler : AsyncRequestHandler<CreateEventManagerInviteCommand>
+    {
+        private AllReadyContext _context;
+
+        public CreateEventManagerInviteCommandHandler(AllReadyContext context)
+        {
+            _context = context;
+        }
+
+        public Func<DateTime> DateTimeUtcNow = () => DateTime.UtcNow;
+
+        protected override async Task HandleCore(CreateEventManagerInviteCommand message)
+        {
+            var @event = await _context.Events.AsNoTracking().SingleOrDefaultAsync(e => e.Id == message.Invite.EventId);
+
+            if (@event == null) throw new ArgumentException("EventId cannot be null for Event manager invite");
```

Comment:
remove this check, we don't do this type of validation in command handlers. we validate in the controller that sends the command. Again, the value message.Invite.EventId is provided by our code and read from our db... so there is no reason that the event would not be found